docs/cockpit.rst

```diff
@@ -0,0 +1,51 @@
+==================================================
+Using the Cockpit with the Atomic Developer Bundle
+==================================================
+
+`Cockpit`_ makes it easy to administer your GNU/Linux servers via a
+web browser.  It can manage system resources and has special functionality
+built in for managing docker containers.  Specifically for containers,
+it can:
+
+* Display a list of images
+* Display a list of containers (started and stopped)
+* Start and stop containers
+* Pull and delete images
+
+.._Cockpit: http://cockpit-project.org/
+
+----------------------------
+Accessing Cockpit in the ADB
+----------------------------
+
+These instructions are written assuming you are at a command prompt on
+the host you are running the ADB on.
+
+1. Start the ADB
+
+2. Log into the ADB
+
+   ``$ vagrant ssh``
+
+3. Start cockpit
+
+   ``$ sudo systemctl start cockpit``
```

Comment:
@bexelbie  You are not doing "systemctl enable docker" in the kickstart file because we do not know how may user would actually use Cockpit? or something else?